plext/using_pljava.html.md.erb

```diff
@@ -179,15 +203,25 @@ Replace \<dbname\> with the name of the target database.
 
 When no databases have PL/Java as a registered language, remove the Java JAR files:
 
-1. Remove the `pljava_classpath` server configuration parameter in the `hawq-site.xml` file.
+1. Remove the `pljava_classpath` server configuration parameter:
```

Comment:
Better to make following procedure two separate procs, one for ambari and one cli.